### PR TITLE
Make 3 edits within Storage concepts

### PR DESCRIPTION
**Pull request by ClimberJ**
I found multiple trivial errors in the Concepts/Storage section of the documentation, which I fixed.

### CODE REVIEW

File: content/en/docs/concepts/storage/projected-volumes.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ This document describes _projected volumes_ in Kubernetes. Familiarity with [vol
 
 ## Introduction
 
-A `projected` volume maps several existing volume sources into the same directory.
+A `projected volume` maps several existing volume sources into the same directory.
```
Comment (sftim) [inline content/en/docs/concepts/storage/projected-volumes.md:19]: The original text is correct; you set the volume type to `projected`, not `projected volume`.

**Review by sftim — COMMENTED** [review]
Thanks. With one fix this would be good to merge.

File: content/en/docs/concepts/storage/projected-volumes.md (hunk 1 above)
Comment (sftim) [reply content/en/docs/concepts/storage/projected-volumes.md:19]: The concept is “projected volumes”; the thing you'd type into a manifest is `projected:` (assuming you're using YAML).

Comment (ClimberJ) [reply content/en/docs/concepts/storage/projected-volumes.md]: The change has been implemented.

**Comment by sftim** [comment]
/retitle Make 3 edits within Storage concepts

**Review by sftim — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**